### PR TITLE
Add 'sic' comment to indicate correctness

### PR DESCRIPTION
**Pull request by dcorderoch**

### CODE REVIEW

**Comment by dcorderoch** [comment]
sorry about the separate PRs, I could join them into one, if you prefer, I'm making them as I go reading the book

**Comment by rrthomas** [comment]
This one is actually correct! Another "% sic" comment would be nice!

And don't worry about the multiple PRs, I'm just happy that readers continue to fix problems!

**Comment by dcorderoch** [comment]
This addresses #121

File: chapters/hpmor-chapter-061.tex [hunk 1]
```diff
@@ -362,6 +362,7 @@
 
 “And they knew about rockets, too?” Severus said dryly. “I don’t believe the other Death Eaters were so fond of Muggle Studies. It is he.”
 
+% sic
```
Comment (rrthomas) [inline chapters/hpmor-chapter-061.tex:365]: @entorb Would it mess up the ebook scripts if there was a comment (and hence line break) in the middle of a paragraph? I would like to be able to have a `% sic` comment directly after the offending word, so it's obvious what's going on.

Comment (entorb) [reply chapters/hpmor-chapter-061.tex:365]: should not be a problem

File: chapters/hpmor-chapter-046.tex [hunk 2]
```diff
@@ -221,6 +221,7 @@
 
 “No,” she finally said. “Please, Harry, don’t ask any more.”
 
+% sic
```
Comment (rrthomas) [inline chapters/hpmor-chapter-046.tex:224]: Please can you put the `% sic` directly after the odd word (almost as if it were in the text, in the conventional way: "this seems wrnog [sic]"?

File: chapters/hpmor-chapter-061.tex (hunk 1 above)
Comment (rrthomas) [reply chapters/hpmor-chapter-061.tex:365]: And again here, please move the `% sic` right after the relevant word.

**Review by rrthomas — COMMENTED** [review]
This is good; please could you move the comments into the paragraphs, just to make it really obvious what is being flagged as "sic"?

**Comment by rrthomas** [comment]
Thanks for working on this, @dcorderoch!

**Pull request merged**